x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/regression_exploration.tsx

```diff
@@ -18,11 +19,22 @@ interface GetDataFrameAnalyticsResponse {
 }
 
 const LoadingPanel: FC = () => (
-  <EuiPanel>
-    <EuiLoadingSpinner className="mlRegressionExploration__evaluateLoadingSpinner" size="xl" />
+  <EuiPanel className="eui-textCenter">
+    <EuiLoadingSpinner size="xl" />
   </EuiPanel>
 );
 
+export const ExplorationTitle: React.SFC<{ jobId: string }> = ({ jobId }) => (
+  <EuiTitle size="xs">
+    <span>
+      {i18n.translate('xpack.ml.dataframe.analytics.regressionExploration.jobIdTitle', {
```

Comment:
`FormattedMessage`? 👀

Reply:
Using FormattedMessage with `EuiTitle` shows an `Types of property 'children' are incompatible` TypeScript error. Since we've got some cleanup to do with replacing other `.translate` with `FormattedMessage` it might be best to update them in a single PR once we have that fix upstream. I can double check with the eui team to see what the timeline for that is.